lib/meta.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Meta
```

Comment:
Should this be in the calculator namespace?

Reply:
I think @blackrat wanted to delete it entirely

Reply:
Yes. `InitializeWithConfig` has been changed to make Meta unnecessary.